administrator/components/com_users/src/Model/MethodModel.php

```diff
@@ -91,7 +91,7 @@ public function methodExists(string $method): bool
     public function getRenderOptions(?User $user = null): SetupRenderOptions
     {
         if (is_null($user)) {
-            $user = Factory::getApplication()->getIdentity() ?: $this->getCurrentUser();
+            $user = $this->getCurrentUser() ?: Factory::getApplication()->getIdentity();
```

Comment:
```suggestion
            $user = $this->getCurrentUser();
```
getCurrentUser automatically falls back to the application